zipline/testing/slippage.py

```diff
@@ -0,0 +1,32 @@
+from zipline.finance.slippage import SlippageModel
+from zipline.utils.sentinel import sentinel
+
+
+class TestingSlippage(SlippageModel):
```

Comment:
We should probably add a test that exercises this if it's going to live in zipline.